features/AuthScreen/SignUpScreen/SignUpScreen.tsx

```diff
@@ -1,41 +1,157 @@
-import React from "react";
-import { View } from "react-native";
-import { Button, Text, TextInput } from "react-native-paper";
+import { useContext, useState } from "react";
+import { Pressable, View } from "react-native";
+import { Button, HelperText, Text, useTheme } from "react-native-paper";
 
 import SafeAreaScreenWrapper from "../../../components/SafeAreaScreenWrapper/SafeAreaScreenWrapper";
 import AuthContext from "../../../contexts/AuthContext";
 import { ScreenType } from "../AuthScreen";
+import CustomTextInput from "../CustomTextInput";
 
 interface SignUpScreenProps {
   screenHandler: (newScreen: ScreenType) => void;
 }
 const SignUpScreen = ({ screenHandler }: SignUpScreenProps) => {
-  const [name, setName] = React.useState("");
-  const [lastName, setLastName] = React.useState("");
-  const [email, setEmail] = React.useState("");
-  const [password, setPassword] = React.useState("");
+  const theme = useTheme();
 
-  const { signUp } = React.useContext(AuthContext);
+  const [name, setName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [displayErrors, setDisplayErrors] = useState(false);
+  const [invalidCredentials, setInvalidCredentials] = useState(false);
+
+  const { signUp } = useContext(AuthContext);
+
+  const nameHandler = (newName: string) => {
+    setName(newName);
+  };
+
+  const nameErrorVisible = () => {
+    return !(name.length > 0);
+  };
+
+  const nameErrorText = () => {
+    return "Name should have at least one character";
```

Comment:
"Name cannot be empty" sounds better for me

Reply:
https://github.com/flatly-pw/mobile-app/pull/11/commits/3e958814e7e18b7b8f7d536cc81367d1ae558143